client/src/components/barchart.js

```diff
@@ -0,0 +1,54 @@
+/** @jsx jsx */
+import React from "react";
+import { jsx } from "@emotion/core";
+import Chart from "chart.js";
+
+function BarChart({ sportsFields }) {
+  React.useEffect(() => {
+    const ctx = document.getElementById("myChart");
```

Comment:
What `ctx` mens?

Reply:
It's just a variable name for the HTML element that is going to contain the chart